includes/class-wc-admin-reports-sync.php

```diff
@@ -22,24 +22,44 @@ class WC_Admin_Reports_Sync {
 	const QUEUE_DEPEDENT_ACTION = 'wc-admin_queue_dependent_action';
 
 	/**
-	 * Action hook for processing a batch of customers.
+	 * Action hook for importing a batch of customers.
 	 */
-	const CUSTOMERS_BATCH_ACTION = 'wc-admin_process_customers_batch';
+	const CUSTOMERS_IMPORT_BATCH_ACTION = 'wc-admin_import_customers_batch';
 
 	/**
-	 * Action hook for processing a batch of orders.
+	 * Action hook for initializing the orders lookup batch creation.
 	 */
-	const ORDERS_BATCH_ACTION = 'wc-admin_process_orders_batch';
+	const CUSTOMERS_DELETE_BATCH_INIT = 'wc-admin_delete_customers_batch_init';
+
+	/**
+	 * Action hook for importing a batch of customers.
+	 */
+	const CUSTOMERS_DELETE_BATCH_ACTION = 'wc-admin_delete_customers_batch';
+
+	/**
+	 * Action hook for importing a batch of orders.
+	 */
+	const ORDERS_IMPORT_BATCH_ACTION = 'wc-admin_import_orders_batch';
 
 	/**
 	 * Action hook for initializing the orders lookup batch creation.
 	 */
-	const ORDERS_LOOKUP_BATCH_INIT = 'wc-admin_orders_lookup_batch_init';
+	const ORDERS_IMPORT_BATCH_INIT = 'wc-admin_orders_lookup_import_batch_init';
 
 	/**
-	 * Action hook for processing a batch of orders.
+	 * Action hook for initializing the orders lookup batch deletion.
 	 */
-	const SINGLE_ORDER_ACTION = 'wc-admin_process_order';
+	const ORDERS_DELETE_BATCH_INIT = 'wc-admin_orders_lookup_delete_batch_init';
+
+	/**
+	 * Action hook for deleting a batch of orders.
+	 */
+	const ORDERS_DELETE_BATCH_ACTION = 'wc-admin_delete_orders_batch';
```

Comment:
Nitpicking here: the two previous action hooks use the wording `orders_lookup` while here it's only `orders`. Is there any reason for this or could it be unified to avoid confusion?